### PR TITLE
Patch 1.3.2

### PR DESCRIPTION
**Pull request by Blade2021**
Updated version control

To change timers, putting machine in to inactive mode is now not required.

### CODE REVIEW

**Comment by Blade2021** [comment]
Patch 1.3.2 has been uploaded to Auto 1, Auto 2, and Auto 3 for testing and verification.

**Comment by Blade2021** [comment]
Resolved conflicting files.

File: MemoryRetrive/MemoryRetrive.ino [hunk 1]
```diff
@@ -70,6 +70,7 @@ void checkData()
       if (apple.substring(0, 7) == "READALL")
       {
         Serial.println("debug 4567");
+
```
Comment (arthus3) [inline MemoryRetrive/MemoryRetrive.ino]: Unnessessary line

File: Auto3/Auto3.ino [hunk 2]
```diff
@@ -1351,57 +1386,158 @@ void eepromUpdate()
   }
 }
 
-int firstValue(){
+void errorReport(byte errorType, int refID)
```
Comment (arthus3) [inline Auto3/Auto3.ino]: What is the purpose of this function?

Comment (Blade2021) [reply Auto3/Auto3.ino]: Function gives a all in one error reporting function for centralization.

**Comment by Blade2021** [comment]
Thank you,  Pull request is set to merge on Monday, April 30th 2018; if testing shows code is without any immediate issues.

**Comment by Blade2021** [comment]
Note: Must test MPS:6 for functionallity and bugs before merge to master branch.

**Comment by Blade2021** [comment]
Issue #25

**Pull request merged**